paywall/src/unlock.js/setupPostOffices.ts

```diff
@@ -14,6 +14,29 @@ interface process {
 
 declare const process: process
 
+export function normalizeConfig(unlockConfig: any) {
+  if (
```

Comment:
this is just filtering out configurations that would crash the lower-casing logic